packages/yoga/src/Button/native/Icon.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import styled, { withTheme } from 'styled-components';
+import { node, oneOfType, func, bool } from 'prop-types';
+import Icon from '../../Icon';
+import withTouchable from './withTouchable';
+
+const ButtonContainer = styled.View`
+  ${({
+    large,
+    secondary,
+    theme: {
+      yoga: {
+        components: { button },
+      },
+    },
+    disabled,
+    inverted,
+    pressed,
+  }) => {
+    const state = secondary ? 'secondary' : 'primary';
+
+    return `
+      display:flex;
+      align-items: center;
+      justify-content: center;
```

Comment:
We usually add the CSS properties which aren't related to the theme or props to be outside the callback

Reply:
commited :)